wpcom-deprecated-functions.php

```diff
@@ -30,11 +30,513 @@ function wpcom_vip_enable_https_canonical() {
+/**
+ * Get the WP.com stats as CSV
+ *
+ * Strings containing double quotes, commas, or "\n" are enclosed in double-quotes. Double-quotes in strings are escaped by inserting another double-quote.
+ * Example: "pet food" recipe
+ * Becomes: """pet food"" recipe"
+ *
+ * @author tott
+ * @deprecated No longer supported since 2.0.0
+ * @param string $table Optional. Table for stats can be views, postviews, referrers, searchterms, clicks. Default is views.
+ * @param string $end_data Optional. The last day of the desired time frame. Format is 'Y-m-d' (e.g. 2007-05-01) and default is UTC date.
+ * @param int $num_days Optional. The length of the desired time frame. Default is 1. Maximum 90 days
+ * @param string $and Optional. Possibility to refine the query with additional AND condition. Usually unused.
+ * @param int $limit Optional. The maximum number of records to return. Default is 5. Maximum 100.
+ * @param bool $summarize Optional. If present, summarizes all matching records.
+ * @return string Result format is CSV with one row per line and column names in first row.
+ */
+function wpcom_vip_get_stats_csv( $table = 'views', $end_date = false, $num_days = 1, $and = '', $limit = 5, $summarize = NULL ) {
+	_deprecated_function( __FUNCTION__, '2.0.0' );
+}
+
+/**
+ * Get the WP.com stats as XML
+ *
+ * @author tott
+ * @deprecated No longer supported since 2.0.0
+ * @param string $table Optional. Table for stats can be views, postviews, referrers, searchterms, clicks. Default is views.
+ * @param string $end_data Optional. The last day of the desired time frame. Format is 'Y-m-d' (e.g. 2007-05-01) and default is UTC date.
+ * @param int $num_days Optional. The length of the desired time frame. Default is 1. Maximum 90 days
+ * @param string $and Optional. Possibility to refine the query with additional AND condition. Usually unused.
+ * @param int $limit Optional. The maximum number of records to return. Default is 5. Maximum 100.
+ * @param bool $summarize Optional. If present, summarizes all matching records.
+ * @return string Result format is XML dataset.
+ */
+function wpcom_vip_get_stats_xml( $table = 'views', $end_date = false, $num_days = 1, $and = '', $limit = 5, $summarize = NULL ) {
+	_deprecated_function( __FUNCTION__, '2.0.0' );
+
+	return null;
+}
+
+/**
+ * Transitionary solution in migration from n to news namespace
+ *
+ * While we get all the VIP clients that have filters that changing the namespace
+ * would impact we will convert n: to news: as late as possible.
+ *
+ * @deprecated No longer supported since 2.0.0
+ * @param mixed $url
+ */
+function wpcom_sitemap_n_to_news_namespace( $url ) {
+	_deprecated_function( __FUNCTION__, '2.0.0' );
+
+	return null;
+}
+
+/**
+ * Helper function for wpcom_vip_load_plugin(); sanitizes plugin folder name.
+ *
+ * You shouldn't use this function.
+ *
+ * @param string $folder Folder name
+ * @return string Sanitized folder name
+ */
+function _wpcom_vip_load_plugin_sanitizer( $folder ) {
```

Comment:
This was rightly moved out of deprecated in https://github.com/Automattic/vip-go-mu-plugins/pull/1088 so we shouldn't include it here.

Reply:
That PR has not been merged yet

Reply:
I missed the part of the PR not yet merged. Regardless of the status of that PR the function is present and loaded as a part of `vip-go-mu-plugins`. If we don't remove it here it's going to fatal when we merge this PR, so I thought it's worth flagging.